smart_proxy_discovery.gemspec

```diff
@@ -18,5 +18,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency('mocha')
   s.add_development_dependency('webmock')
 
+  s.add_runtime_dependency('mime-types', ['= 1.25.1'])
```

Comment:
don't pin it this specifically, as any slight variation won't work (e.g. our packages won't be precisely this)

`< 2.0.0` I reckon.